anyway/flask_app.py

```diff
@@ -381,6 +381,10 @@ def yishuv_symbol_to_name():
     return Response(json.dumps(entries, default=str), mimetype="application/json")
 
 
+source_mapping = {"walla": "וואלה",
```

Comment:
Perhaps adding this to constants.py file?

Reply:
@atalyaalon, thanks for your comment. Totally agree. Fixed and pushed to the PR